src/components/application/application.view.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const ApplicationView: React.FC = () => {
+    return(
+        <>
+            <div className="application">
+                <h3 className="application__header">Demographics</h3>
+                <form className="application__form">
+                <section className="first-last-section">
+                    <div className="application__first-name">
+                        <label className="application__label">First Name</label>
+                        <input id="first-name__input" type="text"/>
```

Comment:
On the inputs, we need to add an `aria-label` and make it `aria-required` since ARIA has a lot of standards covered with browsers. 

```
<input
  type="text"
  aria-label={labelText}
  aria-required="true"
  onChange={onchangeHandler}
  value={inputValue}
  name="name"
/>
```
It's also standard with React to store the input `value` in a stateful variable **as the user is typing**. I personally don't see a case for stateful inputs here so I think it's fine the way it is, but the React docs take it pretty seriously... It's good for scaling features like text recommendations but we don't need that overhead